generated/pom.xml

```diff
@@ -12,6 +12,20 @@
   <artifactId>generated-parent</artifactId>
   <packaging>pom</packaging>
```

Comment:
Ah - possibly the unit test checks are failing now because we made this a multi-module maven project, but did not specify this line `<packaging>pom</packaging>` also in global-autoconfig-properties/pom. Could you try if this is the fix? 

We also probably don't need the dependency block below, since they should be [inherited from parent](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1353#discussion_r1030764468).

Reply:
This did it. Thanks Emily!

Reply:
Makes sense - thanks, Min. 
I changed `global-autoconf...` to look like the datastore pom. Checks are passing locally.